site/_data/cli/shell.yaml

```diff
@@ -119,13 +119,12 @@ commands:
     description: End Position
 - name: recover
   description: Recover the ledger data for failed bookie.
-  argument: [-deleteCookie] <bookieSrc> [bookieDest]
+  argument: <bookieSrc> [<bookieDest>]
   options:
   - flag: -deleteCookie
     description: Delete cookie node for the bookie.
 - name: simpletest
   description: Simple test to create a ledger and write entries to it.
-  argument: [-ensemble N] [-writeQuorum N] [-ackQuorum N] [-numEntries N]
```

Comment:
Is removing the argument intended or incidental? 

Is he problem related to the "[]"? You could try with the multi line syntax for the yank values